ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -1187,8 +1191,8 @@
 		F5E39C7C21F1118D006D65C2 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_STYLE = Automatic;
 				CLANG_ENABLE_CODE_COVERAGE = YES;
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
Nit: Not ideal that CODE_SIGN_STYLE moved.  Perhaps we always get small changes like this and I never noticed.  @wilbaker?

Reply:
I thought that maybe Xcode decided to reorder these when the source files were reordered?

Reply:
Maybe? It's equivalent just noise.
Not sure if it's the move or Xcode version / machine dependent
Not worth blocking over.  I don't recall seeing a change like this previously.

Reply:
It looks like Xcode wants these values to be alphabetized.

When @jrbriggs added `CLANG_STATIC_ANALYZER_MODE` in new spots here:

https://github.com/Microsoft/VFSForGit/commit/c21ad3e5f8fc414104e654970a0704b08d3e51ab#diff-e0c3dea4367db12e2eec645c616be25b

They were ordered alphabetically:

```
CLANG_STATIC_ANALYZER_MODE = deep; 
CODE_SIGN_STYLE = Automatic;
```

And this change is to make the other CLANG_STATIC_ANALYZER_MODE, CODE_SIGN_STYLE entries alphabetical as well.

(It looks like the re-ordered entries were first added here https://github.com/Microsoft/VFSForGit/commit/4342ef86f3fa46cc3197b0a9086cc6f613dfa8c7 and they were not in alphabetical order, I'm not sure if those were added by hand).

I was discussing this with @nickgra offline and he recommended that we not fight Xcode and allow it to re-order things when it wants to.

Reply:
Thanks for verifying @wilbaker that makes sense to me!

Reply:
Sorry, I couldn't stand not having the items in xcode alphabetized :)